telegram/ext/_extbot.py

```diff
@@ -469,6 +469,10 @@ async def _send_message(
         allow_sending_without_reply: ODVInput[bool] = DEFAULT_NONE,
         protect_content: ODVInput[bool] = DEFAULT_NONE,
         message_thread_id: int = None,
+        caption: str = None,
+        parse_mode: ODVInput[str] = DEFAULT_NONE,
+        caption_entities: Union[List["MessageEntity"], Tuple["MessageEntity", ...]] = None,
+        disable_web_page_preview: ODVInput[bool] = DEFAULT_NONE,
```

Comment:
needs to be passed to super as well - that will also fix the failing tests :)

Reply:
I was being sloppy on this occasion, ran only part of the tests locally and left it to CI to crash 🤦🏻

Reply:
Done

Reply:
> I was being sloppy on this occasion, ran only part of the tests locally and left it to CI to crash 🤦🏻 Should at least have checked back sooner to see how many failed

Been there, done that :D